test/test_ops.py

```diff
@@ -1102,6 +1104,27 @@ def test_vjpvmap(self, device, dtype, op):
 
                 self.assertEqual(result_vjps, expected_vjps)
 
+    def _compare_jacobians(self, fn, cotangents_and_primals, argnums=None, atol_rtol=None):
```

Comment:
nit: `_compare_jacobians_of_vjp`? The original name was a bit misleading, that is my bad